packages/server/src/server.ts

```diff
@@ -118,14 +127,16 @@ const logAndHandleResponse = async (methodName, methodFunction) => {
   const messagePrefix = `[POST] ${methodName}:`;
   try {
     const response = await methodFunction();
-    const success = response instanceof JsonRpcError ? 'Failure' : 'Success';
+    const success = response instanceof JsonRpcError ? responseFailureLabel : responseSuccessLabel;
     ms = Date.now() - start;
-    methodSuccessLatencyCounter.inc({ method: methodName, success: success, latency: ms });
+    methodSuccessLatencyCounter.inc({ method: methodName, success: success });
+    methodLatencyHistogram.labels(methodName, success).observe(ms);
     logger.info(`${messagePrefix} ${ms} ms ${success}`);
     return response;
   } catch (e) {
     ms = Date.now() - start;
-    methodSuccessLatencyCounter.inc({ method: methodName, success: 'Failure', latency: ms });
+    methodSuccessLatencyCounter.inc({ method: methodName, success: responseFailureLabel });
```

Comment:
Is this handling RPC errors like the error codes mentioned [here](https://deploy-preview-482--hedera-hips.netlify.app/hip/hip-482#errors)? Those should be considered as unsuccessful for metrics purposes.

Reply:
Updated to capture code though